src/sentry/testutils/outbox.py

```diff
@@ -32,3 +41,30 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
     with TaskRunner(), assume_test_silo_mode(SiloMode.MONOLITH):
         while enqueue_outbox_jobs():
             pass
+
+
+def assert_webhook_outboxes(
+    factory_request: WSGIRequest,
+    webhook_identifier: WebhookProviderIdentifier,
+    region_names: List[str],
+):
+    expected_payload = ControlOutbox.get_webhook_payload_from_request(request=factory_request)
+    expected_payload_dict = dataclasses.asdict(expected_payload)
+    region_names_set = set(region_names)
+
+    if ControlOutbox.objects.count() != len(region_names_set):
+        raise Exception("Found more ControlOutboxes than provided region_names")
+    assert ControlOutbox.objects.count() == len(region_names)
```

Comment:
Aren't these redundant with each other?

Reply:
Yep, meant to remove the assertion in favour of the error.